docs/proposals/observability-management.md

```diff
@@ -0,0 +1,133 @@
+* [Observability Management](#observability-management)
```

Comment:
Proposal is already in another PR, please remove this part in this PR.

Reply:
Done